core/server/master/src/main/java/alluxio/master/file/NoopBlockDeletionContext.java

```diff
@@ -19,11 +19,9 @@ public final class NoopBlockDeletionContext implements BlockDeletionContext {
 
   @Override
   public void registerBlockForDeletion(long blockId) {
-    return;
```

Comment:
typically we add a comment explicitly saying it is a noop, not that we forgot to implement.

Reply:
will add comment